apps/web/src/components/SkillsInDetail/SkillsInDetail.stories.tsx

```diff
@@ -1,55 +1,37 @@
 import React from "react";
 import { action } from "@storybook/addon-actions";
-import { Meta, Story } from "@storybook/react";
+import { StoryFn } from "@storybook/react";
 
 import { fakeSkills } from "@gc-digital-talent/fake-data";
-import { BasicForm, Submit } from "@gc-digital-talent/forms";
+import { BasicForm } from "@gc-digital-talent/forms";
 
 import type { FormSkills } from "~/types/experience";
 
-import SkillsInDetail, { SkillsInDetailProps } from "./SkillsInDetail";
+import SkillsInDetail from "./SkillsInDetail";
 
 export default {
   component: SkillsInDetail,
   title: "Components/Skills In Detail",
   args: {
     skills: [],
-    handleDelete: action("Remove from experience"),
+    onDelete: (skillId: string) =>
+      action("Remove skill from experience")(skillId),
   },
-} as Meta;
+};
 
-const TemplateSkillsInDetail: Story<SkillsInDetailProps> = (args) => {
-  const { skills } = args;
+const Template: StoryFn<typeof SkillsInDetail> = (args) => {
   return (
     <BasicForm onSubmit={action("submit")}>
       <SkillsInDetail {...args} />
-      {skills.length !== 0 && <Submit />}
     </BasicForm>
   );
 };
 
-export const NoSkills = TemplateSkillsInDetail.bind({});
-export const FewSkills = TemplateSkillsInDetail.bind({});
-export const ManySkills = TemplateSkillsInDetail.bind({});
-
-NoSkills.args = {
-  skills: [],
-};
-
-FewSkills.args = {
+export const Default = Template.bind({});
+Default.args = {
   skills: fakeSkills(2).map((skill) => ({
```

Comment:
This could be 1 skill: any reason that we need to render multiple?

Reply:
There is some base vertical spacing built in so we should keep render more than 1 to confirm it is intact.